### PR TITLE
Introduce Chapter-3 Introduction Page

### LINKED ISSUE #501: Create Intro page for Chapter 3
Page 1:
image

Page 2:
image

Should follow the same conventions as intros for chapter 1 and 2

### PR DESCRIPTION
**Pull request by shaavan**
Closes #501
- This PR introduces the Intro page and introduces the subsequent updates to progress to keep track of them.
- This PR also temporarily introduces challenge-1 for Chapter-3, which is introduced to check the correct functioning of the "Continue" button. It will be removed before the final merge of PR.

### CODE REVIEW

**Comment by afterburn** [comment]
@shaavan retargeted your branch to `dev`, please resolve conflicts when you have time, thanks

**Comment by shaavan** [comment]
Updated from 70f79e9a115807b777557d1171dae4f55eca24ae to ff3d94c2f8f92c53f3f3618f47b94bcfd765c829 ([pr516.01](https://github.com/shaavan/saving-satoshi/commits/pr516.01) -> [pr516.02](https://github.com/shaavan/saving-satoshi/commits/pr516.02), [diff](https://github.com/shaavan/saving-satoshi/compare/pr516.01..pr516.02))
Address @afterburn comment

Changes:
1. Rebased over `dev`
2. Resolved merge conflicts

File: content/lessons/chapter-3/index.ts [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import * as Intro1 from './intro-1'
+import * as Intro2 from './intro-2'
+import * as YVB1 from './you-vs-bit-1'
+
+const Chapter3 = {
+  'intro-1': Intro1,
+  'intro-2': Intro2,
+  'you-vs-bit-1': YVB1,
```
Comment (afterburn) [inline content/lessons/chapter-3/index.ts:8]: Lets rename `you-vs-bit-1` to `solo-1` and for the co-op and pool once we'll add later we can use `coop` and `pool`

File: content/lessons/chapter-3/you-vs-bit-1.tsx [hunk 2]
```diff
@@ -0,0 +1,22 @@
+'use client'
+
+import { useTranslations } from 'hooks'
+import { Introduction, Text } from 'ui'
+
+export const metadata = {
+  title: 'chapter_two.intro_one.title',
+  image: '/assets/images/chapter-2-intro-1.jpg',
+  key: 'CH3YVB1',
```
Comment (afterburn) [inline content/lessons/chapter-3/you-vs-bit-1.tsx:9]: Keys can be `CH3SOL1`, `CH3COO1`, `CH3POO1` (that last one is a but unfortunate lol)

**Review by afterburn — CHANGES_REQUESTED** [review]
Thanks @shaavan, requested some minor changes.

File: content/lessons/chapter-3/index.ts (hunk 1 above)
Comment (shaavan) [reply content/lessons/chapter-3/index.ts:8]: Sounds good! Let me rename it!

**Comment by shaavan** [comment]
Updated from ff3d94c2f8f92c53f3f3618f47b94bcfd765c829 to 5a1178b3952e7d024452fad6416a2a65c9fdba47 ([pr516.02](https://github.com/shaavan/saving-satoshi/commits/pr516.02) -> [pr516.03](https://github.com/shaavan/saving-satoshi/commits/pr516.03), [diff](https://github.com/shaavan/saving-satoshi/compare/pr516.02..pr516.03))
Addressed @afterburn comment

- Renamed `you-vs-bit-1` -> `solo-1`
- Renamed `YVB1` -> `SOL1`

File: content/lessons/chapter-3/index.ts (hunk 1 above)
Comment (shaavan) [reply content/lessons/chapter-3/index.ts]: Addressed in 5a1178b3952e7d024452fad6416a2a65c9fdba47
Thanks for the suggestion!

File: content/lessons/chapter-3/you-vs-bit-1.tsx (hunk 2 above)
Comment (shaavan) [reply content/lessons/chapter-3/you-vs-bit-1.tsx]: Addressed in https://github.com/saving-satoshi/saving-satoshi/commit/5a1178b3952e7d024452fad6416a2a65c9fdba47

**Review by afterburn — APPROVED** [review]

**Pull request merged**